mapadroid/webhook/webhookworker.py

```diff
@@ -408,20 +415,15 @@ def __prepare_mon_data(self, mon_data):
             if self.__is_in_excluded_area([mon["latitude"], mon["longitude"]]):
                 continue
 
-            if not self.__args.pokemon_webhook_nonivs \
-               and mon["pokemon_id"] in self.__IV_MON \
-               and (mon["individual_attack"] is None):
-                # skipping this mon since IV has not been scanned yet
-                continue
-
             mon_payload = {
-                "encounter_id": mon["encounter_id"],
+                "encounter_id": str(mon["encounter_id"]),
```

Comment:
What's the reason to make this a string?

Reply:
Jabes said JS has issues working with encounter IDs since their too big. (according to him that also means it's invalid json)

rdm also sends them as Str and so does spezus' pvp plugin. It doesn't break PA or Poracle.

Reply:
My request - this number is too large for this to be valid json. This causes parsing problems in javascript - JSON.parse loses precision resulting in cut-off id numbers in poracle logs confusing users.
RDM has always been a string so precedent, and confirmed with other tool vendors no problem (and real world testing with this PR shows no issue)

Reply:
Understood, thanks